package.json

```diff
@@ -318,5 +318,8 @@
   },
   "peerDependencies": {
     "react-hook-form": "^7.0.0"
+  },
+  "dependencies": {
+    "caniuse-lite": "^1.0.30001698"
```

Comment:
we need this?

Reply:
Is this comment resolved? I couldn't find any reference to this package in the diff.